README.md

```diff
@@ -0,0 +1,8 @@
+# Scylla-CLI
+
+Add some description here
+
+## Installation
+```shell
+pip install -r requirements.txt
+```
```

Comment:
@benipeled please add a new line
so the file will be easier to maintain later on.

Otherwise looks good, thanks!

Reply:
Well, you right,
but we might want to switch it to `virtualenv` in order to avoid deps issues - WDYT?:

```shell
python -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
```